src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -133,29 +138,28 @@ Template:
-| T1 Rho map         | T1rho            | Quantitative T1rho brain imaging <br> <https://www.ncbi.nlm.nih.gov/pubmed/24474423> <br> <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4346383/> |
-| T1 map             | T1map            | quantitative T1 map                                                                                                                               |
-| T2 map             | T2map            | quantitative T2 map                                                                                                                               |
-| T2\*               | T2star           | High resolution T2\* image                                                                                                                        |
-| FLAIR              | FLAIR            |                                                                                                                                                   |
-| FLASH              | FLASH            |                                                                                                                                                   |
-| Proton density     | PD               |                                                                                                                                                   |
-| Proton density map | PDmap            |                                                                                                                                                   |
-| Combined PD/T2     | PDT2             |                                                                                                                                                   |
-| Inplane T1         | inplaneT1        | T1-weighted anatomical image matched to functional acquisition                                                                                    |
-| Inplane T2         | inplaneT2        | T2-weighted anatomical image matched to functional acquisition                                                                                    |
-| Angiography        | angio            |                                                                                                                                                   |
+non-parametric structural MR images include:
+
+| **Name**                                   | `suffix`         | **Description**                                                                                                                                   |
+| ------------------------------------------ | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| T1 weighted images                         | T1w              | Structural images with predominant T1 contribution.                                                                                               |
```

Comment:
It seems like a waste to just say T1 here (and T2, PD, etc. below) when you say what the quantitative version means in much more detail below. I'm no physicist, but would it be reasonable to say "Structural images predominantly weighted based on longitudinal relaxation time."

Reply:
It might be worth it to be explicit and include "In arbitrary units (arbitrary)." in this table to match the convention of the next table.

Reply:
@tsalo agreed that we can be more informative here. Please see https://github.com/bids-standard/bids-specification/pull/690/commits/dce708d2e8b9ac518546cd3674798644802d1848. I hope that this is not an over-kill.

Reply:
@tsalo what do you think?

Reply:
I think it looks much better now. Minor note though- arbitrary is misspelled as abitrary in the updated table.

Sorry for the delay in responding. I lost track of this thread somehow.

Reply:
Thank you for catching that @tsalo! b4f1570